addons/mail/static/src/core/common/im_status.xml

```diff
@@ -17,7 +17,7 @@
                     <t t-if="(!props.member or !props.member.isTyping) and persona">
                         <i t-if="persona.im_status === 'online'" class="fa fa-circle text-success" title="Online" role="img" aria-label="User is online"/>
                         <i t-elif="persona.im_status === 'away'" class="fa fa-circle o-away" title="Idle" role="img" aria-label="User is idle"/>
-                        <i t-elif="persona.im_status === 'offline'" class="fa fa-circle-o text-700 opacity-75" title="Offline" role="img" aria-label="User is offline"/>
+                        <i t-elif="persona.im_status === 'offline'" class="fa fa-circle-o text-muted" title="Offline" role="img" aria-label="User is offline"/>
```

Comment:
As it's not only `700` to `body` in this commit, you should probably specify this change in the description as well :)

Reply:
_btw the globe icon is too light too, this reminds me I should fix it too)_

Reply:
Reverted the change: https://github.com/odoo/odoo/compare/b311263adf344774f05d70366a15cd3b35998c42..69481a272ff63dbcd4400f399a72a92db1588631